### DISCUSSION TITLE
Discussões sobre a Linguagem Egua

### DISCUSSION THREAD

**Discussion by lucaspompeun** [eguatech/egua#17] [General]
Fala, pessoal. Beleza?
Passando pra dizer que aqui será o canal de comunicação direta entre todos. xD

**Comment by trtecnologia** [comment]
Gostaria de deixar um comentário de parabéns ainda não testei mas logo irei fazer testes, gostei de toda a postagem feita numa página chamada terminalroot que me trouxe até esse projeto, espero que continuem, e gostaria também de deixar meus cumprimentos a você lucas e a todo mundo que estiver empenhado, apesar do nome ter sido inusitado, é a demonstração de que nós Brasileiros temos de nos orgulhar de tudo que fazemos aqui. Novamente PARABÉNS!

**Reply by lucaspompeun** [reply]
O nome é inusitado, mas o significado carrega uma cultura imensa <3. Obrigado pelas palavras, fico feliz demais de ver que mais pessoas estão conhecendo a linguagem

**Comment by ghost** [comment]


**Reply by ItaloCobains** [reply]
Se vc achou interessante de um olhada em delegua tbm. https://github.com/DesignLiquido/delegua

**Comment by scoobiii** [comment]
Parabens Lucas e time!
Explorando novas ideias p/ o ecossistema Égua:

✅ LLM integrada à linguagem Égua
✅ IDE com Égua LLM
✅ Framework Égua

LLM geradora de código respeitando parser, lexer e interpreter. Rumo ao full stack Égua sem bugs!
#dev #LLM #ÉguaLang

🔗 https://joslghpk.manus.space/

**Reply by scoobiii** [reply]
https://qyzurugm.manus.space/roteiro
